networks/suzuka/setup/src/local.rs

```diff
@@ -6,6 +6,7 @@ use dot_movement::DotMovement;
 
 #[derive(Debug, Clone, Default)]
 pub struct Local {
+	// this is just for the current version without settlement because this field is never read.
```

Comment:
Might just slap `#[allow(dead_code)]` on it and keep it private, so we can grep for these warnings overrides later.